src/components/molecules/ChatPoll/ChatPoll.scss

```diff
@@ -78,7 +78,7 @@ button.ChatPoll__question {
       }
     }
 
-    &-background {
+    &__background {
```

Comment:
you've changed the classname here but not in the component. What's the desired behavior?

Reply:
reverted the changed in 4932822, was faulty change